apps/src/gamelab/GameLabSprite.js

```diff
@@ -105,6 +108,58 @@ module.exports.createSprite = function (x, y, width, height) {
     }
   });
 
+  Object.defineProperty(s, 'width', {
+    enumerable: true,
+    get: function () {
+      if (width === undefined) {
+        return 100;
+      } else if (!s.animation) {
+        return unscaledWidth;
+      } else {
+        console.log("old width: " + width);
+        return unscaledWidth * this.scale;
+      }
+    },
+    set: function (value) {
+      unscaledWidth = value / this.scale;
```

Comment:
Alright, I think I figured this part out too. We will need a dependency on animation in our setters, but right now our width/height are completely ignored when drawing animations so it doesn't matter (we just use the width/height of the animation image and don't end up resetting our width/height properties to match the image). 

I think I'll have to make that bug fix in this PR too so we don't end up in any broken states. Right now it prints out the correct width/height, but it won't change how the animation is drawn.
https://www.pivotaltracker.com/projects/1521049/stories/121709415